specification/postgresql/Microsoft.OrionDB.Management/examples/2025-08-01-preview/Clusters_CreateOrUpdate.json

```diff
@@ -0,0 +1,203 @@
+{
+  "operationId": "Clusters_CreateOrUpdate",
```

Comment:
We don't have these operationId and title in the examples on Flex. Are we missing those there or shouldn't these two be here?

Reply:
In typespec it is required in order to map examples to the auto-genration openApi.json file. I think there is no harm on adding it